src/cf-typography/usage.md

```diff
@@ -57,18 +121,20 @@
 
 ### Large pull quote
 
-<aside class="pull-quote pull-quote__large">
-    <div class="pull-quote_body">
-        Lorem ipsum dolor sit amet, consectetur adipisicing elit.
-        Cum corrupti tempora nam nihil qui mollitia consectetur
-        corporis nemo culpa dolorum!
-    </div>
-    <footer>
-        <cite class="pull-quote_citation">
-            - Author Name
-        </cite>
-    <footer>
-</aside>
+<div class="block">
+    <aside class="pull-quote pull-quote__large">
+        <div class="pull-quote_body">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            Cum corrupti tempora nam nihil qui mollitia consectetur
+            corporis nemo culpa dolorum!
+        </div>
+        <footer>
+            <cite class="pull-quote_citation">
+                - Author Name
+            </cite>
+        </footer>
+    </aside>
+</div>
```

Comment:
I was playing with that earlier and decided against it. I'm going to play with some ideas to have some padding and borders around code samples like in USWDS. I'll search for old blocks that made it through.

https://standards.usa.gov/typography/